pkgs/tools/system/chase/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl ,pkgconfig, libatomic_ops , boehmgc }:
+
+stdenv.mkDerivation rec {
+  name = "chase_0.5.2";
```

Comment:
The file on the debian repo seperates name and version with a "_" , so i used it .
Thank you.